1.0/genBinaryMC.sh

```diff
@@ -0,0 +1,17 @@
+# adjust according to actual installation path of gltf-pipeline
+alias gltfpipe="node /home/user/ehome/projects/gltf/gltf-pipeline-master/bin/gltf-pipeline.js"
```

Comment:
Maybe use an npm dependency instead of a user-specific path here?

Reply:
I was following the style of the existing convertAll.sh script. 
Perhaps just remove the script to avoid confusion ?
Since I am unsure how to use an npm dependency here, I just commented the alias and added an explanation.

Reply:
The script is bit more complicated/strange because gltfpipe insisted on its 'output' directory.